_blogs/2015-02-10-hello-world.md

```diff
@@ -0,0 +1,5 @@
+---
+title: Hello World
```

Comment:
Is this required for production?

Reply:
addressed in [this questioned](https://github.com/bsadd/bsaddhome/pull/9#discussion_r426148408)